### PR TITLE
Portfolio: Mobile Menu Milestone

### PR DESCRIPTION
**Pull request by mateoburvi**
# Add Mobile Menu for Portfolio project

In this part of the portfolio, we add:

- CSS for the active menu 
- JS for the interactions on the menu 

Note: I don't have a coding Partner, that's why it doesn't have commits from him 

Coding  Partner absence

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    // set  menu state
+    showMenu = true;
+  } else {
+    header.classList.remove('header-active');
+    header.classList.add('header');
+    burger.classList.remove('burger-active');
+    burger.classList.add('burger');
+    document.body.style.overflowY = 'auto';
+
+    // Set menu state
```
Comment (alexander16108) [inline main.js:24]: hello there currently when i click on the hamburger menu it doesn't behave as expected. it currently doesn't work at all please fix this little bug and try again 😉

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @mateoburvi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no contribution from a coding partner 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @mateoburvi 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — COMMENTED** [review]
Hi @mateoburvi  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved2


## Highlights
Good PR description
No linter errors
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**